src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -5693,6 +5801,19 @@ instruct countTrailingZerosL_mem(rRegI dst, memory src, rFlagsReg cr) %{
   ins_pipe(ialu_reg_mem);
 %}
 
+instruct countTrailingZerosL_mem_nf(rRegI dst, memory src, rFlagsReg cr) %{
+  predicate(UseAPX && UseCountTrailingZerosInstruction);
+  match(Set dst (CountTrailingZerosL (LoadL src)));
+  effect(KILL cr);
+
+  ins_cost(175);
```

Comment:
Could you please let me know why you're using ins_cost here and not in above pattern?